pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceTest.java

```diff
@@ -979,7 +980,31 @@ public void testLookupThrottlingForClientByClient() throws Exception {
         EventLoopGroup eventLoop = EventLoopUtil.newEventLoopGroup(20, false,
                 new DefaultThreadFactory("test-pool", Thread.currentThread().isDaemon()));
         long reqId = 0xdeadbeef;
-        try (ConnectionPool pool = new ConnectionPool(conf, eventLoop)) {
+
+        // using an array in order to reset a new CountDownLatch
+        CountDownLatch[] latch = new CountDownLatch[1];
```

Comment:
Is there has thread safety issue with multiple threads accessing `CountDownLatch[] latch`? Now the code should be fine, I feel like maybe 'AtomicReference' is better

Reply:
`AtomicReference` is better. Have updated. @poorbarcode